CMakeLists.txt

```diff
@@ -100,6 +100,12 @@ IF(BUILD_WITH_URDF_SUPPORT)
   LIST(APPEND CFLAGS_DEPENDENCIES "-DPINOCCHIO_WITH_URDFDOM")
 ENDIF(BUILD_WITH_URDF_SUPPORT)
 
+IF(${URDFDOM_VERSION} VERSION_GREATER "0.4.2")
+  SET(CMAKE_CXX_STANDARD 11)
```

Comment:
Which platforms are still based on urdfdom < 1.0?

Reply:
at least 16.04

Reply:
So it's fair to say that from 18.04 onwards, Pinocchio will essentially be C++11 by default (URDF is on by default I believe).

Reply:
Yes.